app/src/main/java/com/example/akademiaandroida/features/location/presentation/model/LocationDisplayable.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.akademiaandroida.features.location.presentation.model
+
+
+data class Location(
```

Comment:
Można tutaj zaznaczyć, że jest to klasa `LocationDisplayable`. Wtedy są mniejsze szanse o pomyłkę (ta sama nazwa nie występuję dwa razy).